CUE4Parse/UE4/Assets/Exports/SkeletalMesh/FSkelMeshSection.cs

```diff
@@ -268,7 +268,7 @@ public void SerializeRenderItem(FAssetArchive Ar)
                 Ar.Position += 4;
                 break;
             case EGame.GAME_CalabiYau or EGame.GAME_FragPunk:
-                Ar.Position += 8;
+                Ar.Position += 14;
```

Comment:
This needs to be validated to ensure compatibility with FragPunk as well

Reply:
sorry, i do not have FragPunk ,so i split the change to two statements